hudi-utilities/src/test/java/org/apache/hudi/utilities/UtilitiesTestBase.java

```diff
@@ -193,19 +204,60 @@ public static void saveStringsToDFS(String[] lines, FileSystem fs, String target
       os.close();
     }
 
+    /**
+     * Converts the json records into CSV format and writes to a file.
+     *
+     * @param hasHeader  whether the CSV file should have a header line.
+     * @param sep  the column separator to use.
+     * @param lines  the records in JSON format.
+     * @param fs  {@link FileSystem} instance.
+     * @param targetPath  File path.
+     * @throws IOException
+     */
+    public static void saveCsvToDFS(
+        boolean hasHeader, char sep,
+        String[] lines, FileSystem fs, String targetPath) throws IOException {
+      Builder csvSchemaBuilder = CsvSchema.builder();
+
+      ArrayNode arrayNode = mapper.createArrayNode();
+      Arrays.stream(lines).forEachOrdered(
+          line -> {
+            try {
+              arrayNode.add(mapper.readValue(line, ObjectNode.class));
+            } catch (IOException e) {
+              e.printStackTrace();
```

Comment:
don't we need to throw here as well?

Reply:
This should not happen though but agree that we can throw exception here to catch any conversion issues.  Note that this is only used in the test code.